packages/web/src/components/admin_page/feature_switch_page/row_switch.jsx

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+import { OverlayTrigger, Popover, Button, Form, Alert } from 'react-bootstrap';
+import { useState } from 'react';
+import { parseToNewAPI, featureSwitchesURL } from '../../../Global';
+import axios from 'axios';
+import { FsContext } from '../../../feature_switches';
+import { updateSwitches } from './render_switch';
+import { useContext } from 'react';
+
+const popover = (onDelete, onUpdate, errorMessage) => {
+    return (
+        <Popover id="popover-basic">
+            <Popover.Header as="h3">Want to update or remove?</Popover.Header>
+            <Popover.Body>
+                {errorMessage !== undefined ? <Alert variant="warning">{errorMessage}</Alert> : null}
+                <UpdateSwitch onUpdate={onUpdate} />
+                <Button variant="danger" onClick={onDelete}>
+                    Remove
+                </Button>
+            </Popover.Body>
+        </Popover>
+    );
+};
+
+const HEADERS = { 'BEARER-TOKEN': localStorage.getItem('auth_token') };
```

Comment:
Don't we have the headers defined in Global.js? If not define them there and import here.

Reply:
I think we do not I will add it and can check it when i commit